src/radical/pilot/session.py

```diff
@@ -714,6 +714,8 @@ def _get_resource_sandbox(self, pilot):
                     for k,v in pilot['description'].items():
                         if v is None:
                             v = ''
+                        if k == 'project' and '_' in v and 'ornl' in resource:
+                            v = v.split('_')[0]
```

Comment:
What is the purpose of this change?

Reply:
this part was moved from outer scope, since it is related to PD attributes it is under `if '%' in sandbox_raw:` statement now, and it doesn't interfere with such case as having `_` in env variable name (`$MEMBERWORK/$LSB_PROJECT_NAME`)